addons/spectrum/functions/fnc_spectrumDeviceMouseDown.sqf

```diff
@@ -51,16 +51,20 @@ if (count _frequenciesSorted > 0) then {
 	private _strongestSignal = _frequenciesSorted#0;
 	
 	// if type is "radioChatter", play sound and save beacon in gvar
-	_unit = _strongestSignal#0;
-	_range = _strongestSignal#2;
-	_type = _strongestSignal#3;
+	private _unit = _strongestSignal#0;
+	private _frequency = _strongestSignal#1;
+	private _range = _strongestSignal#2;
+	private _type = _strongestSignal#3;
+	private _sigStrength = [_unit, player, _range] call FUNC(calcSignalStrength);
+
+	// broadcast local event that we are now "active" on this signal
+	[QGVAR(activatedSpektrumDevice), [_type, _frequency, _unit, _sigStrength]] call CBA_fnc_localEvent;
```

Comment:
`activatedSpektrumDevice` should probably be
`activatedSpectrumDevice` (with a `c` instead of a `k`)

Reply:
Good catch. Thanks Bernard! The danish spelling sneaked its way in :sweat_smile: